cpp/include/cudf/quantiles.hpp

```diff
@@ -42,5 +42,36 @@ quantile(column_view const& input,
          interpolation interp = interpolation::LINEAR,
          order_info column_order = {});
 
+/* @brief Gathers the rows corrosponding to the desired quantiles.
```

Comment:
```suggestion
/** @brief Returns the rows of the input corresponding to the requested quantiles.
```

(note `/**` for doxygen)